update/ansible/playbook/tasks/files/maintenance.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Maintenance Mode</title>
+</head>
+<body>
+<h1>We're Currently upgrading PMM</h1>
+<p>We apologize for the inconvenience. Please check back later.</p>
```

Comment:
```suggestion
<h1>PMM is being upgraded</h1>
<p>Please wait until the upgrade process is finished.</p>
```

Reply:
I believe it's in the user's interest to upgrade. Moreover, it's the user, who triggers the upgrade, so I'd not apologize much but rather ask them for patience.

Reply:
We probably need to refresh this page every N seconds, otherwise the user wouldn't know when the upgrade is finished.

Here is a piece of html to insert into the page head, that should refresh the page every 5 secs.

`<meta http-equiv="refresh" content="5" />`

Reply:
Admin upgrades PMM, this page will be visible for everybody else.